frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectImportPortlet.java

```diff
@@ -128,12 +131,12 @@ private boolean isImportSuccessful(ImportStatus importStatus) {
         return (importStatus.isSetRequestStatus() && importStatus.getRequestStatus().equals(RequestStatus.SUCCESS) && importStatus.getFailedIds().isEmpty());
     }
 
-    private ImportStatus importDatasources(List<String> toImport, User user, RemoteCredentials remoteCredentials)  {
+    private ImportStatus importDatasources(List<String> toImport, User user, RemoteCredentials remoteCredentials) {
         ImportStatus importStatus = new ImportStatus();
         try {
             importStatus = bdpImportClient.importDatasources(toImport, user, remoteCredentials);
             if (!isImportSuccessful(importStatus)) {
```

Comment:
if the thrift file could be renamed this could be also renamed at the same occasion.